.travis.yml

```diff
@@ -1,19 +1,12 @@
 language: go
 go:
-  - release
-  - tip
+  - stable
+  - master
 sudo: false
-before_install:
-  - gotools=golang.org/x/tools
-  - if [ "$TRAVIS_GO_VERSION" = "release" ] || [ "$TRAVIS_GO_VERSION" = "go1.3.3" ]; then gotools=code.google.com/p/go.tools; fi
 install:
   - go get -d -t -v ./...
-  - go get -v $gotools/cmd/cover
-  - go get -v $gotools/cmd/vet
-  - go get -v github.com/bradfitz/goimports
-  - go get -v github.com/golang/lint/golint
```

Comment:
What's the motivation for changing these?

Reply:
- `$gotools` trick is for Go < 1.4, so we probably don't need it anymore 
- I believe `vet` and `cover` tools are pre-installed by default now
- `github.com/bradfitz/goimports` moved to `golang.org/x/tools/cmd/goimports`
- `github.com/golang/lint/golint` moved to `golang.org/x/lint/golint`